geopandas/io/tests/test_arrow.py

```diff
@@ -535,7 +539,7 @@ def test_parquet_missing_metadata(tmpdir, naturalearth_lowres):
 
     # missing metadata will raise ValueError
     with pytest.raises(
-        ValueError, match="Missing geo metadata in Parquet/Feather file."
+        ValueError, match=re.escape("Missing geo metadata in Parquet/Feather file.")
```

Comment:
```suggestion
        ValueError, match="Missing geo metadata in Parquet/Feather file"
```

Similar? (and several cases below)